.evergreen/serverless/setup-secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# setup-secrets.sh
+#
+# Set up serverless secrets using Drivers AWS Secrets Manager.
+# Writes a secrets-export.sh file to this folder.
+set -eux
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+. $SCRIPT_DIR/../handle-paths.sh
+pushd $SCRIPT_DIR
```

Comment:
Should there be a matching `popd` at the end of this script?

Reply:
Done